src/microbe_stage/organelle_unlocks/WorldBasedUnlockCondition.cs

```diff
@@ -144,17 +146,46 @@ public override void GenerateTooltip(LocalizedStringBuilder builder, IUnlockStat
         {
             var compoundName = Compound!.InternalName;
 
-            if (Min.HasValue && Max.HasValue)
+            var percentageFormat = TranslationServer.Translate("PERCENTAGE_VALUE");
+            var unitFormat = TranslationServer.Translate("VALUE_WITH_UNIT");
+
+            string? formattedMin;
+            string? formattedMax;
+
+            switch (compoundName)
+            {
+                case "sunlight":
+                    formattedMin = Min.HasValue ? unitFormat.FormatSafe(percentageFormat.FormatSafe(Min), "lx") : null;
```

Comment:
```suggestion
                    formattedMin = Min.HasValue ? new LocalizedString("PERCENTAGE_VALUE", new LocalizedString("PERCENTAGE_VALUE", Min)), "lx") : null;
```
Something like that.

Reply:
I'm not sure I fully understand, but storing a LocalizedString in a temporary variable might make the design easier to do?